ledger/participant-state/kvutils/app/src/main/resources/com/daml/ledger/participant/state/kvutils/app/logback.base.xml

```diff
@@ -15,6 +15,19 @@
         <appender-ref ref="STDOUT"/>
     </root>
 
+    <!-- Disable noisy DB logging at the start of sandbox -->
+    <logger name="com.digitalasset.platform.store.FlywayMigrations" level="WARN" />
+    <logger name="org.flywaydb" level="ERROR" />
+    <logger name="com.zaxxer.hikari" level="ERROR" />
+    <logger name="com.digitalasset.platform" level="WARN" />
+
+    <!-- raising the command tracker logging level -->
+    <logger name="com.digitalasset.platform.apiserver.services.tracking.TrackerMap" level="WARN" />
+
+    <!--
+    explicitly enabling the info logging for sandbox main and ledger api server
```

Comment:
I don't see how this comment relates to the lines below. Can you clarify this, please?

Reply:
Thanks. Fixed.